### PR TITLE
updated DTD/XSL

### PR DESCRIPTION
**Pull request by himorin**
to match with recent /TR/ spec template update, contents as https://github.com/w3c/ttml2/pull/1224/files

for https://github.com/w3c/ttwg/issues/181

### CODE REVIEW

File: spec/xmlspec-ttml1.xsl [hunk 1]
```diff
@@ -612,6 +608,44 @@ width: 100%;
...
+
+<!-- ghrepoloc: participate section -->
+<!-- called in a <dl> content from header -->
+<xsl:template match="ghrepoloc">
+  <xsl:apply-templates/>
+</xsl:template>
+<xsl:template match="ghloc">
+  <dt>Participate:</dt>
+  <dd><a href="https://github.com/{@repo}/">GitHub <xsl:value-of select="@repo"/></a></dd>
+  <dd><a href="https://github.com/{@repo}/issues/">File a bug</a></dd>
```
Comment (nigelmegitt) [inline spec/xmlspec-ttml1.xsl]: This goes to the issues list not the "create an issue" page - is that intentional?

File: spec/xmlspec-ttml1.xsl [hunk 2]
```diff
@@ -612,6 +608,44 @@ width: 100%;
...
+  <dd><a href="https://github.com/{@repo}/commits/{@branch}">Commit history</a></dd>
+  <dd><a href="https://github.com/{@repo}/pulls/">Pull requests</a></dd>
```
Comment (nigelmegitt) [inline spec/xmlspec-ttml1.xsl]: This is too much for my tastes: it takes 4 lines that all basically take the reader to the GitHub repo. Is there a reason we need the lines other than the GitHub repo link?

**Review by nigelmegitt — COMMENTED** [review]
Verified that the CR Snapshot change to entitiescr.dtd works.

File: spec/xmlspec-ttml1.xsl (hunk 1 above)
Comment (himorin) [reply spec/xmlspec-ttml1.xsl]: lines are from what respec inserts (e.g. https://w3c.github.io/mediacapture-main/), and I'm not sure what was the original intent...
Actually, bikeshed inserts a link pointing issues/new, so might be better to raise this to respec also?

Comment (nigelmegitt) [reply spec/xmlspec-ttml1.xsl]: Makes sense to have consistency, yes. Anyway I think we can go straight for issues/new since the text says "File a bug".

**Review by nigelmegitt — APPROVED** [review]

**Pull request merged**